src/plugins/data/server/search/opensearch_search/opensearch_search_strategy.test.ts

```diff
@@ -146,7 +146,7 @@ describe('OpenSearch search strategy', () => {
         return new DataSourceError({});
       },
       dataSourceEnabled: jest.fn(() => true),
-      defaultClusterEnabled: jest.fn(() => false),
+      hideLocalCluster: jest.fn(() => true),
```

Comment:
Can we change the text of test case as well? e.g. `dataSource enabled and local cluster is hidden, send request with dataSourceId get data source client`.

Basically change from `default cluster disabled` to `local cluster is hidden' in this test file for better readability.

Reply:
Test description updated. Thanks for the comment~